web/src/containers/Trade/components/OrdersWrapper.js

```diff
@@ -73,7 +73,7 @@ class OrdersWrapper extends Component {
 		const USER_TABS = [
 			{
 				stringId: 'ORDERS',
-				title: STRINGS['ORDERS'],
+				title: STRINGS['ORDERS'] + " (" + activeOrders.length + ")",
```

Comment:
I think it's better to use template literals instead to be more consistent with the codebase. Could you please change that @genievot